modules/kubernetes-role-assignment.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-cluster-configuration.adoc
+:_module-type: PROCEDURE
+[id="kubernetes-role-assignment_{context}"]
+= Finding Kubernetes roles are their assignment
+
+[role="_abstract"]
+Use the *Configuration Management* view to get information about which Kubernetes roles are assigned to which users and groups.
+
+.Procedure
+. Navigate to the RHACS portal and click *Configuration Management* from the left-hand navigation menu.
```

Comment:
After reading through a few more of the PRs, I realise that the product name "Red Hat Advanced Cluster Security for Kubernetes" is always referred to in full in the docs and the portal for the product is always referred to as "the RHACS portal".

In hindsight, you could consider referring to the full name of the portal the first time you use it and then use the acronym for further references, in each module.

Please see [this related conversation in another PR](https://github.com/openshift/openshift-docs/pull/34199/files#r669543422).

Reply:
For this PR I have added the acronym in the assembly. I'll updated all assemblies to always have that acronym appear along with first mention of the name so that I don't have to include full name in all modules.